_layouts/topic.html

```diff
@@ -126,7 +126,7 @@ <h2 id="maintainers">Maintainers</h2>
         {% for maintainer in topic.maintainers %}
             <li>
             {% if contributors[maintainer].github != false %}
-                <a href="https://github.com/{{ maintainer }}">{{ contributors[maintainer].name }}</a>
+                <a href="/hall-of-fame#{{ maintainer }}">{{ contributors[maintainer].name }}</a>
             {% else %}
                 {{ contributors[maintainer].name }}
```

Comment:
Now you could add a link to the `/hall-of-fame` of frame for people without GitHub account. What do you think?

Reply:
oh yes, good catch, thanks :)